cime_config/config_tests.xml

```diff
@@ -193,7 +193,11 @@ LII    CLM initial condition interpolation test
     <BFBFLAG>TRUE</BFBFLAG>
     <STOP_OPTION>ndays</STOP_OPTION>
     <STOP_N>11</STOP_N>
+    <HIST_OPTION>$STOP_OPTION</HIST_OPTION>
+    <HIST_N>$STOP_N</HIST_N>
+    <CONTINUE_RUN>FALSE</CONTINUE_RUN>
     <DOUT_S>FALSE</DOUT_S>
+    <SMP_BUILD>0</SMP_BUILD>
```

Comment:
Actually if the initial case has threads=2 then case 2 needs to be built with OMP enabled or it will fail.